index.js

```diff
@@ -0,0 +1,144 @@
+  + 'sold around the world.'
++ '</p>'
++ '<ul class="project-stacks card-stacks">'
+  + '<li>SLICK</li>'
+  + '<li>HIGH QUALITY</li>'
+  + '<li>LIGHT</li>'
+  + '<li>Camera Hi</li>'
+  + '<li>Interactive</li>'
++ '</ul>';
+  return innerElem;
+}
+
+const cardelem1 = card(['card', 'weather'], details('white-description'));
+const cardelem2 = card(['card', 'gtime'], details('white-description'));
+const cardelem3 = card(['card', 'conference'], details('white-description'));
+const cardelem4 = card(['card', 'katchy'], details('white-description'));
+
+const cardlists = document.createElement('div');
+cardlists.classList.add('cards-list');
+
+[cardelem1, cardelem2, cardelem3, cardelem4].forEach((item) => item.appendChild(cardlists));
+
+const aboutussection = document.createElement('section');
+aboutussection.setAttribute('id', 'about-me-section');
+aboutussection.innerHTML = '<h2>About us</h2> <p class="about-me-text">'
+  + 'We are skilled in providing high end mobile technology for the world with'
+  + 'beautiful design. We have adapted best screen technologies through years. <br />'
+  + 'We also give consultabcy for other tech companies to develop their technology!'
+  + '<br /> if you\'d like to get in contact about purchases, please reach out! </p>'
++ '<a href="https://docs.google.com/document/d/1lYq0Kb50TujzJzDWGobIchRI17JMnAiZMF9OBQgNU68/edit?usp=sharing"'
+  + 'class="link" target="_blank" rel="noreferrer noopener"> Get our Catalog </a> <img'
+  + 'class="about-me-icon mobile-icon" src="./media/svg/icon-2.svg" alt="Icon" />'
++ '<img class="about-me-icon desktop-icon" src="./media/svg/desktop-icon-2.svg" alt="Icon"/>';
+
+const aboutus = document.createElement('div');
+aboutus.setAttribute('id', 'about-me');
+aboutus.appendChild(aboutussection);
+
+const cardsh2 = document.createElement('h2');
+cardsh2.innerText = 'Recent Releases';
+const firstcard = document.createElement('div');
+firstcard.classList.add('first-card');
+firstcard.innerHTML = '<img src="./media/png/ubrunning.png" alt="UB-Running Screenshot" class="first-card-img"/>'
++ '<div class="first-card-description">'
+    + '<h2>Techno F20</h2><p class="description blue-color">'
+      + 'A Mobile made with the lastest screen technology and good UX</p>'
++ '<ul class="project-stacks blue-color"> <li>LIGHT</li> <li>SLICK</li> <li>Hi Screen</li>'
+      + '<li>Hi Networg</li> <li>Image Graphics</li> <li>Battery</li> </ul>'
+    + '<a href="https://github.com/Spayco/UB-Running/" class="link" target="_blank" '
+    + 'rel="noreferrer noopener" >See project</a> </div>';
+
+const project = document.createElement('div');
+project.setAttribute('id', 'projects');
+[cardsh2, firstcard, cardelem1, cardelem2, cardelem3,
+  cardelem4].forEach((item) => project.appendChild(item));
+
+const main = document.createElement('main');
+[featured, project, aboutus].forEach((item) => main.appendChild(item));
+
+[header, main].forEach((item) => document.body.appendChild(item));
```

Comment:
- [ ] You should avoid using `innerHTML`. You can use `textContent` instead of it. 

 The use of innerHTML creates a potential security risk for your website. Malicious users can use cross-site scripting (XSS) to add malicious client-side scripts that steal private user information stored in session cookies.